### DISCUSSION TITLE
Allow ChangeLog Site Example to be Resizable

### DISCUSSION THREAD

**Discussion by dhurlburtusa** [olivierlacan/keep-a-changelog#486] [Ideas]
At https://keepachangelog.com/en/1.0.0/, the example changelog is great. However, I think it would be better to make it resizable so that a user can see more of it if they choose.

Just need to add the `resize` property to `.changelog` in `source/assets/stylesheets/sections.sass`.
```sass
div.header
  .changelog
    resize: both;
```

Please see PR #293.

**Workaround**

Open Dev Tools, find the element, add `resize: both`.
On site, use resize handle to expand changelog.

**Comment by olivierlacan** [comment]
Hey @dhurlburtusa. I'm not sure I want the example changelog to be freely resizable horizontally and vertically. As it is, even vertically the resizing messes with the header background and frankly doesn't provide much beyond what scrolling the fixed size preview currently does. I think I'd be more interested in a link to the full file in the repository instead for people who want to see the shape of the whole document. That said I'm a little concerned providing an external link this high up on the page might mess with the flow of the page a bit. 

I appreciate your work on #293 but for now I'm going to hold on and think about how to achieve what you're looking for slightly differently. Feel free to suggest anything else though. I don't mean to shut you down. 😃

**Comment by dhurlburtusa** [comment]
@olivierlacan No problem.

Another idea would be to put it in a `textarea`. That way a user could set focus on it (tabbing, mouse click, etc), select-all (`Ctrl-A`), and copy somewhere (`Ctrl-C`). With the `pre` tag, it is just a little more difficult to select the changelog text.

Note: Will need to add some CSS to remove the default border around the textarea -- may need some other styling too.

BTW, I just tried adding `tabindex="1"` to the `pre` tag. It allows it to be focused but `Ctrl-A` still selects the text content of the entire page.

BTW, thanks for starting this project so I can have something to refer other developers to and to reference myself!

**Comment by olivierlacan** [comment]
@dhurlburtusa That's an interesting idea. Sorry for not getting back earlier? Would you mind updating your PR to just allow vertical resizing and try out the textarea idea?

**Comment by NicoHood** [comment]
@dhurlburtusa What about the textarea patch? This sounds correct to me and makes sense, if it can be resized vertically.
